test/fixers/test_crystal_format_fixer_callback.vader

```diff
@@ -0,0 +1,33 @@
+Before:
+  Save g:ale_crystal_format_executable
+  Save g:ale_crystal_format_options
+
+  " Use an invalid global executable, so we don't match it.
+  let g:ale_crystal_format_executable = 'xxxinvalid'
+  let g:ale_crystal_format_options = ''
+
+  call ale#test#SetDirectory('/testplugin/test/fixers')
+
+After:
+  Restore
+
+  call ale#test#RestoreDirectory()
+
+Execute(The crystal format callback should return the correct default values):
+
+  AssertEqual
+  \ {
+  \   'command': ale#Escape('xxxinvalid') . ' format -',
+  \ },
+  \ ale#fixers#crystal#Fix(bufnr(''))
+
+Execute(The crystal format callback should include custom options):
+  let g:ale_crystal_format_options = "-list=true"
+
+  AssertEqual
```

Comment:
This test is falling with:

```
 (2/2) [EXECUTE] (X) {'command': 'xxxinvalid tool format -list=true -'} should be equal to {'command': 'xxxinvalid -list=true -'}
```

Again make sure the test matches exactly the return value of `ale#fixers#crystal#Fix()`.